stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/internal/MessageListScrollHelper.kt

```diff
@@ -42,6 +44,19 @@ internal class MessageListScrollHelper(
         }
         recyclerView.addOnScrollListener(
             object : RecyclerView.OnScrollListener() {
+
+                /**
+                 * Checks if we currently have a popup shown over the list.
+                 *
+                 * @param recyclerView The list that we're observing.
+                 * @param newState The scroll state of the list.
+                 */
+                override fun onScrollStateChanged(recyclerView: RecyclerView, newState: Int) {
```

Comment:
I still don't get why long click listener returning `true` doesn't consume the click.

Reply:
Yeah I'm not 100% sure about that either. We can play around with more research, maybe we can find a better solution in the future